docs/administration.index.md

```diff
@@ -28,45 +28,47 @@ See the [meshWorkspace](meshcloud.workspace.md) documentation for details about
+| &nbsp;&nbsp;[Delete&nbsp;Tenants](administration.delete-tenants.md#delete-tenants)                                  |       &#10003;       |       &#10003;       |     &#10003;      |  &#10003;   |            |      &#10003;      |                    |                      |
+| &nbsp;&nbsp;&nbsp;&nbsp;[History](administration.projects.md#project-history)                                       |       &#10003;       |       &#10003;       |     &#10003;      |  &#10003;   |            |      &#10003;      |                    |                      |
+| &nbsp;&nbsp;[Chargeback&nbsp;Statements](administration.usage.md)                                                   |       &#10003;       |       &#10003;       |                   |             |  &#10003;  |                    |                    |                      |
+| [Platforms](administration.platforms.md)                                                                            |       &#10003;       |       &#10003;       |                   |             |            |                    |                    |                      |
+| &nbsp;&nbsp;[Platform&nbsp;Notifications](administration.platforms.md#platform-notifications)                       |       &#10003;       |       &#10003;       |     &#10003;      |             |            |                    |                    |                      |
+| &nbsp;&nbsp;[Platform&nbsp;Restrictions](administration.platforms.md#restrict-platform-access)                      |       &#10003;       |       &#10003;       |     &#10003;      |  &#10003;   |            |                    |                    |                      |
+| &nbsp;&nbsp;[Landing&nbsp;Zones](administration.landing-zones.md)                                                   |       &#10003;       |       &#10003;       |     &#10003;      |             |            |                    |                    |                      |
+| &nbsp;&nbsp;[Usage&nbsp;Reports](administration.usage.md)                                                           |       &#10003;       |       &#10003;       |                   |             |  &#10003;  |                    |                    |                      |
+| &nbsp;&nbsp;[Tenants](administration.tenants.md)                                                                    |       &#10003;       |       &#10003;       |     &#10003;      |  &#10003;   |            |                    |                    |         &#10003;     |
+| &nbsp;&nbsp;View [Unmanaged Tenants](administration.unmanaged-tenants.md)                                           |       &#10003;       |       &#10003;       |     &#10003;      |             |            |                    |                    |         &#10003;     |
```

Comment:
Not sure why the diff is so messed up but I only added these two lines.

Also it's still TBD who can import unmanaged tenants

Reply:
only Partner Admin can assign the tenant to project.
Other than that everything is fine, except maybe for the new term  we use